deploy/images.csv

```diff
@@ -9,6 +9,7 @@ KUBEVIRT_LIBGUESTFS_TOOLS_IMAGE,quay.io/kubevirt/libguestfs-tools,KUBEVIRT_VERSI
 KUBEVIRT_EXPORTPROXY_IMAGE,quay.io/kubevirt/virt-exportproxy,KUBEVIRT_VERSION,1017c0e7097bf309e7a2e8542656f76c3ee1365ff03d01f563dbe108a98153b1
 KUBEVIRT_EXPORSERVER_IMAGE,quay.io/kubevirt/virt-exportserver,KUBEVIRT_VERSION,10d35daa56ec1981bbbcf898f5200909e044b3ae34a3f790ca82fedc906c52d9
 NETWORK_PASST_BINDING_IMAGE,quay.io/kubevirt/network-passt-binding,KUBEVIRT_VERSION,e6e01ca7adedb6215caa7284c8e38ee5afa9ff9033180907d6027941b2477f0a
+NETWORK_PASST_BINDING_CNI_IMAGE,quay.io/oshoval/network-passt-binding-cni,KUBEVIRT_VERSION,237dfdc49476a2477665faf2b74262ed8821f31a6da261eb78883eaa2134716c
```

Comment:
dont care about the hacks right now, it is because there isnt release yet with it
there are more hacks, hence it is wip and partial, i.e it doesnt have all the unit tests / doesnt deploy all required objs yet
it is just so we can sync about the direction so far please
thanks

Reply:
it is in kubevirt, it is a hack please disregard it for now
it will be on quay.io/kubevirt/network-passt-binding-cni
once there is a new version 1.6
exactly as was done for quay.io/kubevirt/network-passt-binding (line above)
on https://github.com/kubevirt/hyperconverged-cluster-operator/pull/3034

we wont merge it before the image is available